unitTests/ui/QuerySuggestPreviewTest.ts

```diff
@@ -102,7 +103,8 @@ export function QuerySuggestPreviewTest() {
         const numberOfPreviewResults = 5;
         setupQuerySuggestPreview({ numberOfPreviewResults });
         await triggerPopulateSearchResultPreviewsAndPassTime();
-        expect(test.cmp.queryController.getLastQuery().numberOfResults).toBe(numberOfPreviewResults);
+        const lastSearchQuery = (test.cmp.queryController.getEndpoint().search as jasmine.Spy).calls.mostRecent().args[0] as IQuery;
+        expect(lastSearchQuery.numberOfResults).toEqual(numberOfPreviewResults);
```

Comment:
I would add tests or assertions in an existing test checking that `searchHub`, `pipeline`, `tab` etc. are being included in the query.

Reply:
Good idea, I'll add those tests